src/main/java/seedu/tp/commands/ReviewedCommand.java

```diff
@@ -18,8 +19,9 @@ public ReviewedCommand (FlashcardList flashcardList, int index, Ui ui) {
     @Override
     public void execute() throws InvalidFlashcardIndexException {
         try {
-            flashcardList.getFlashcardAtIdx(index).setReviewStatus(true);
-            ui.confirmFlashcardReview(flashcardList.getFlashcardAtIdx(index));
+            Flashcard flashcard = flashcardList.getFlashcardAtIdx(index);
```

Comment:
In ShowCommand it's index - 1. shouldn't they both be the same?

Reply:
Other than this I think everything looks good.
Add some Javadoc comments so the build passes, then I think it's good for merging

Reply:
> In ShowCommand it's index - 1. shouldn't they both be the same?

I'm thinking it might be better if we deal with the `-1` when we initialize all the commands. Like for `DeleteCommand` we have `return new DeleteCommand(flashcardList, Integer.parseInt(splitInput[1]) - 1);` in `Parser.java`, so maybe we can change `ShowCommand` to ` return new ShowCommand(flashcardList, Integer.parseInt(splitInput[1]), ui);` and same for `ReviewedCommand`.

Reply:
Noted. I did that so that I didn't have to +1 when printing the index, but for consistency I agree it's better to deal with -1 in the Parser itself.